public/views/index/body.ejs

```diff
@@ -13,6 +13,7 @@
                             <a class="lasuite-header__service-link ui-home" href="/" title="Accueil - Notepad de l'État" aria-label="Accueil - Notepad de l'État">
```

Comment:
Just noticed the `a` wrapping this has its label overridden for screen reader users - make sure they also get the info (aria-label overrides the element content).

```suggestion
                            <a class="lasuite-header__service-link ui-home" href="/" title="Accueil - Notepad de l'État" aria-label="Accueil - Notepad de l'État (beta)">
```